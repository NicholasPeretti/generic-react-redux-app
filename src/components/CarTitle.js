import React from 'react'
import PropTypes from 'prop-types'

var CarTitle = ({ manufacturerName, modelName }) => (
  <React.Fragment>
    {manufacturerName} {modelName}
  </React.Fragment>
)

CarTitle.propTypes = {
  manufacturerName: PropTypes.string.isRequired,
  modelName: PropTypes.string.isRequired
}

export default CarTitle
