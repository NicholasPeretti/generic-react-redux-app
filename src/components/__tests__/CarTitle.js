import React from 'react'
import { shallow } from 'enzyme'
import PresentationalTest from '../../utils/tests/presentational'
import CarTitle from '../CarTitle'
const props = {
  manufacturerName: 'Fiat',
  modelName: 'Marea'
}

describe('CarTitle', () => {
  PresentationalTest(CarTitle, props)

  const wrapper = shallow(<CarTitle {...props}/>)

  test('Should match snapshot', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
