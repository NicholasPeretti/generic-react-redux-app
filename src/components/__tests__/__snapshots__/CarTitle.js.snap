// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CarTitle Should match snapshot 1`] = `"<h3 style=\\"font-weight:bold\\" class=\\"sc-bdVaJa fkyhBf\\">Fiat Marea</h3>"`;
